zkvm/src/blockchain/errors.rs

```diff
@@ -0,0 +1,34 @@
+use crate::errors::VMError;
+
+#[derive(Debug, Fail)]
+pub enum BCError {
```

Comment:
maybe this longer name is not bad, so we avoid clunky abbreviations? That’s my personal preference though.
```suggestion
pub enum BlockchainError {
```